.github/workflows/samples-python-fastapi-server.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
-          python-version: '3.9'
+          python-version: '3.12'
```

Comment:
3.9 is the minimal supported version. shall we keep it as it's to ensure changes won't break 3.9 support?

Reply:
Is this 3.9 or 3.8 actually?

Reply:
@wing328 I used 3.8 instead of 3.7 ; I couldn't find the information whether the generator was supporting 3.8 or 3.9.

As I say in the first post, 3.8 will be out of support next month ; I'm happy to remove 3.8 in favor of 3.9 as well.